crates/libs/bindgen/src/standalone.rs

```diff
@@ -19,6 +19,7 @@ pub fn standalone(names: &[&str]) -> String {
 
     let mut type_names = BTreeSet::new();
     let mut core_types = BTreeSet::new();
+    let mut enums = BTreeMap::new();
```

Comment:
@kennykerr I think you might have meant this to be a `Vec`? Otherwise this will overwrite the entry instead of adding to it:
```rust
enums.insert(enum_name, type_name.name);
```

Reply:
Ah, good catch - will fix asap.

Reply:
Here's the fix: #2434 - I use sorted containers to ensure that the generated code is reproducible.

Reply:
Great, thanks!